test/core/tsi/alts/fake_handshaker/fake_handshaker_server.cc

```diff
@@ -58,15 +58,12 @@ namespace gcp {
 // It is thread-safe.
 class FakeHandshakerService : public HandshakerService::Service {
  public:
-  FakeHandshakerService(int expected_max_concurrent_rpcs,
-                        const std::string& peer_identity)
-      : expected_max_concurrent_rpcs_(expected_max_concurrent_rpcs),
-        peer_identity_(peer_identity) {}
+  FakeHandshakerService(const std::string& peer_identity)
```

Comment:
Single-arg ctors need to be `explicit`.

https://google.github.io/styleguide/cppguide.html#Implicit_Conversions

Reply:
done